pallets/collator-selection/README.md

```diff
@@ -0,0 +1 @@
+License: Apache-2.0
```

Comment:
Hmm was  under the impression everything was gpl 3

Reply:
This pallet has been removed.